test/insecure/configuration.spec.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+/* eslint-env mocha */
+
+const chai = require('chai')
+chai.use(require('dirty-chai'))
+chai.use(require('chai-as-promised'))
+const { expect } = chai
+
+const Transport = require('libp2p-websockets')
+const { NOISE: Crypto } = require('libp2p-noise')
```

Comment:
This is in the `insecure` folder but is using noise. Did you intend to use plaintext for this? If not this should just get moved into core for now

Reply:
I am moving it to core instead